src/main/java/model/LottoStatistics.java

```diff
@@ -1,35 +1,42 @@
 package model;
 
-import constant.WinLottoInfo;
+import java.util.Arrays;
 import java.util.EnumMap;
 import java.util.List;
 import java.util.Map.Entry;
+import java.util.Set;
 
 public class LottoStatistics {
-    private final EnumMap<WinLottoInfo, Integer> lottoStatistics = new EnumMap<>(WinLottoInfo.class);
+    private final EnumMap<LottoWinRank, Integer> lottoStatistics = new EnumMap<>(LottoWinRank.class);
 
     public LottoStatistics(List<LottoNumbers> lottoNumbers, WinLotto winLotto) {
+        Arrays.stream(LottoWinRank.values()).forEach(lottoWinRank -> lottoStatistics.put(lottoWinRank, 0));
         for (LottoNumbers purchasedLotto : lottoNumbers) {
-            WinLottoInfo winResult = WinLottoInfo.result(purchasedLotto, winLotto);
-            lottoStatistics.put(winResult, lottoStatistics.getOrDefault(winResult, 0) + 1);
+            LottoWinRank winResult = LottoWinRank.calculateLottoWinRank(purchasedLotto, winLotto);
+            lottoStatistics.put(winResult, lottoStatistics.get(winResult) + 1);
         }
+        lottoStatistics.remove(LottoWinRank.NONE);
+    }
+
+    public Integer getCount(LottoWinRank lottoWinRank) {
+        return lottoStatistics.get(lottoWinRank);
+    }
+
+    public Double totalReturn(Integer purchaseAmount) {
+        return (double) calculateTotalPrize() / purchaseAmount;
+    }
```

Comment:
```suggestion
    public double totalReturn(Integer purchaseAmount) {
        return (double) calculateTotalPrize() / purchaseAmount;
    }
```
원시타입으로 캐스팅했는데 래퍼타입을 반환할 필요가 있ㅇ르까요?

Reply:
엇 그러네요 이 부분 수정하면서 `calculateTotalPrize()`의 반환값도 `int`로 변경했습니다!
8cb4e57